<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>文件发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>359</y>
      <width>500</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>200</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="msgList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>500</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="porEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="portLb">
       <property name="text">
        <string>PORT:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ipLb">
       <property name="text">
        <string>IP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ipEdit"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="coonBtn">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
